client/my-sites/plugins/main.jsx

```diff
@@ -348,18 +351,8 @@ const PluginsMain = React.createClass( {
 			);
 		}
 
-		if ( this.state.accessError ) {
-			return (
-				<Main>
-					{ this.renderDocumentHead() }
-					<SidebarNavigation />
-					<EmptyContent { ...this.state.accessError } />
-					{ this.state.accessError.featureExample
-						? <FeatureExample>{ this.state.accessError.featureExample }</FeatureExample>
-						: null
-					}
-				</Main>
-			);
+		if ( ! this.props.userCanManagePlugins ) {
```

Comment:
This change introduces a regression - if you try to load `/plugins/:site` with a clean Redux state, you'll see the following:

![](https://cldup.com/uUKXlLKwQo.png)

While in production, nothing is being displayed while loading.

It would probably make sense to display a placeholder instead of nothing while the sites are loading, but not a generic 404 error message.